include/inja/environment.hpp

```diff
@@ -105,7 +105,7 @@ class Environment {
 
   Template parse(std::string_view input) {
     Parser parser(parser_config, lexer_config, template_storage, function_storage);
-    return parser.parse(input, input_path);
+    return *parser.parse(input, input_path);
```

Comment:
we are doing parse just on config time, right? if so i'm ok with this copy.

Reply:
Yes, we are only parsing at config time. I will add a comment to the area where this is called in envoy-gloo to explicitly call out that we should only be parsing at config time.